pkg/backend/filestate/testdata/Pulumi.yaml

```diff
@@ -0,0 +1,3 @@
+name: test-project
```

Comment:
Do we need this test project today?

Reply:
It's currently used by the tests in `pkg/backend/filestate/backend_test.go`